pom.xml

```
@@ -24,13 +24,13 @@
     <module>jrebel</module>
   </modules>
 
-  <url>http://stapler.kohsuke.org/</url>
+  <url>https://stapler.kohsuke.org/</url>
 
   <licenses>
     <license>
       <name>2-clause BSD license</name>
       <distribution>repo</distribution>
-      <url>http://opensource.org/licenses/bsd-license.php</url>
+      <url>https://opensource.org/licenses/bsd-license</url>
```

```suggestion
      <url>https://opensource.org/licenses/BSD-2-Clause</url>
```

seems to be canonical, or is at least what is linked to from https://opensource.org/licenses.

---

I've seen the first one a number of places, but the second should be goot.